### PR TITLE
Fix lab1 assertion dense layer output assertion

### PR DESCRIPTION
**Pull request by prashantkhurana**
Fix lab1 assertion dense layer output assertion :
1) tf.random.set_seed only sets the global seed https://www.tensorflow.org/api_docs/python/tf/random/set_seed 

2) we need to set keras set_random_seed to make all seeds be deterministic. 

docs : https://www.tensorflow.org/api_docs/python/tf/config/experimental/enable_op_determinism, https://www.tensorflow.org/api_docs/python/tf/keras/utils/set_random_seed.enable_op_determinism might not be needed here but its good to have.

### CODE REVIEW

**Review by OnigiriJack — APPROVED** [review]

**Review by dominikhaska — APPROVED** [review]

**Review by jacobhq — APPROVED** [review]
Thanks, this worked for me!
